concordium-consensus/src/Concordium/MultiVersion.hs

```diff
@@ -1001,46 +1001,79 @@ sendCatchUpStatus genIndex = MVR $ \mvr@MultiVersionRunner{..} -> do
 -- simply rely on catch-up.
 withLatestExpectedVersion ::
     GenesisIndex ->
-    (EVersionedConfiguration gsconf finconf -> MVR gsconf finconf UpdateResult) ->
-    MVR gsconf finconf UpdateResult
+    (EVersionedConfiguration gsconf finconf -> MVR gsconf finconf (Skov.UpdateResult, Maybe a)) ->
+    MVR gsconf finconf (Skov.UpdateResult, Maybe a)
 withLatestExpectedVersion gi a = do
     vvec <- liftIO . readIORef =<< asks mvVersions
     -- Length is an Int and GenesisIndex is a Word32.
     -- Assuming a 64-bit system, there is no risk of over/underflow.
     case compare (Vec.length vvec - 1) (fromIntegral gi) of
         EQ -> a (Vec.last vvec)
-        LT -> return ResultInvalidGenesisIndex
-        GT -> return ResultConsensusShutDown
+        LT -> return (Skov.ResultInvalidGenesisIndex, Nothing)
+        GT -> return (Skov.ResultConsensusShutDown, Nothing)
+
+withLatestExpectedVersion_ ::
```

Comment:
I feel like I had a comment here that said that `'` has no information content, and now there is `_` which similarly does not.

Can you at least document it

Reply:
Yes. I will. Naming of these things are kind of hard..